plugins/Violet Magic theme/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+	"author": "Fluffy hyena",
+	"name": "Violet Magic Theme",
+	"version": "1.0.0",
+	"id": "d4a11ede-ec96-49af-ab3c-f0f3857d8d8d",
+	"description": "Adds a nice looking theme to bridge (looks better in dark theme)",
+	"api_version": 1,
+	"tags": [
+		"Theme"
+	],
+	"releaseTimestamp": 1613568421816
```

Comment:
Please remove the `releaseTimestamp` from your manifest

Reply:
Done